etna/ciim/client.py

```diff
@@ -164,19 +157,17 @@ def search(
         *,
         q: Optional[str] = None,
         web_reference: Optional[str] = None,
-        start_date: Optional[datetime] = None,
-        end_date: Optional[datetime] = None,
-        date_field: Optional[DateField] = None,
+        opening_start_date: Optional[datetime] = None,
+        opening_end_date: Optional[datetime] = None,
```

Comment:
If the client expects a `datetime`, but the user only selects a `date` - we'll probably need to turn the 'end dates' into 'end of the day' datetimes to have the desired affect (e.g. `2021-03-30-23:59:99`). This will apply to both `opening_end_date` and `closing_end_date`

Reply:
> If the client expects a `datetime`, but the user only selects a `date` - we'll probably need to turn the 'end dates' into 'end of the day' datetimes to have the desired affect (e.g. `2021-03-30-23:59:99`). This will apply to both `opening_end_date` and `closing_end_date`

Nice one Andy. Agree, but current ticket scope is for the parameter changes, thought to address this in another ticket DF-209 where date values are input individually.

Reply:
Sensible :) Works for me!